.github/workflows/nox.reference.languages.cd.yaml

```diff
@@ -1,23 +0,0 @@
-name: Publish
-
-on:
-  release:
-    types: [released]
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        dotnet-version: ['6.0.x']
```

Comment:
don't delete pls

Reply:
Why? Languages is part of Nox.Reference.Data.World so it belongs to World workflow

Reply:
But why then we have .github/workflows/nox.reference.phonenumbers.ci.yaml?

Reply:
Should we either have it and languages or none of them?

Reply:
Phone numbers and holidays will be moved to World as well. So far these are different projects workflows still exist